components/formattedLevelInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useContext, useState } from 'react';
+import Level from '../models/db/level';
+import { LevelContext } from '../contexts/levelContext';
+import Record from '../models/db/record';
+import getFormattedDate from '../helpers/getFormattedDate';
+import useStats from '../hooks/useStats';
+
+interface RecordDivProps {
+  record: Record;
+}
+
+function RecordDiv({ record }: RecordDivProps) {
+  return (<>
+    <br/>
+    <span className='font-bold'>{record.moves}</span>
+    <span> by </span>
+    <span className='font-bold'>{record.userId.name}</span>
+    <span> - {getFormattedDate(record.ts)}</span>
+  </>);
+}
+
+interface FormattedLevelInfoProps {
+  level: Level;
+}
+
+export default function FormattedLevelInfo({ level }: FormattedLevelInfoProps) {
+  const [copied, setCopied] = useState(false);
+  const levelContext = useContext(LevelContext);
+  const { stats } = useStats();
+  const stat = stats?.find(stat => stat.levelId === level._id);
+
+  const recordDivs = [];
+
+  if (levelContext?.records) {
+    for (let i = 0; i < levelContext.records.length; i++) {
+      recordDivs.push(<RecordDiv key={i} record={levelContext.records[i]} />);
+    }
+  }
+
+  return (
+    <div>
```

Comment:
Again i would consider seeing if there was a card component that could work here from tail wind to help keep consistent look and feel

Reply:
Going to leave this for now but come back to it in a future PR. For now this is mainly copying over the existing styles from the modals into the sidebar.